Micr.cs

```diff
@@ -124,7 +130,7 @@ public static bool IsValid( string content )
                     return false;
                 }
 
-                if ( micr.GetAccountNumber().Length < 6 )
+                if ( string.IsNullOrWhiteSpace( micr.GetOnUs() ) )
```

Comment:
Shouldn't be needed, already checked by checking if GetAccountNumber is less than 6 characters.